update/upgrade_db.sh

```diff
@@ -35,10 +36,11 @@ case $key in
     -h|--help)
         echo "Arguments:"
         echo -e "\t-c|--clean\t\tCleans comp and test DB before starting"
-        echo -e "\t-d|--localdirpath\tAdds local customization delta files"
+        echo -e "\t-d|--initdirs\tAdds local customization init files"
```

Comment:
I would find it better for the user to have a `-e|--extdirs` argument. So the user would do:

```shell
$ upgrade_db.sh -e /path/qwat-data-model/extensions/ch_vd_sire /path/to/my/customization1 /path/to/my/customization2
```

But this would mean standardizing on the extension/customization init script name (`init.sh`), and therefore renaming the ch_vd_sire init script from `init_sire.sh` to `init.sh`.

Don't you think that that would be more intuitive to the user?